test/fuzz/validated_input_generator.cc

```diff
@@ -427,13 +427,12 @@ void ValidatedInputGenerator::onEnterMessage(Protobuf::Message& msg,
                                !reflection->HasOneof(msg, descriptor->oneof_decl(oneof_index)))) {
       // No required member in one of set, so create one.
       for (int index = 0; index < oneof_desc->field_count(); ++index) {
-        const std::string parents_class_name = parents.back()->GetDescriptor()->full_name();
+        const std::string class_name = descriptor->full_name();
```

Comment:
parent name is also used in another function `handleAnyRules`

Just curious is that the expected behavior?

Reply:
Yes, there it is correct. The name of the `msg` class is `google.protobuf.Any` which gives us no clue which typed_configs are valid there. Therefore looking at the parent's class name is what we want to learn what is a valid typed_config. Thanks for spotting, but this is all ok!